ui/components/app/confirmations/alerts/multiple-alert-modal/index.scss

```diff
@@ -0,0 +1,26 @@
+@use "design-system";
+
+.multiple-alert-modal {
+  &__arrow-buttons {
```

Comment:
Have we confirmed these navigation arrows aren't already in use elsewhere in which case the styles may already exist?